SimTracker/TrackAssociation/plugins/TrackTimeValueMapProducer.cc

```diff
@@ -66,7 +66,7 @@ class TrackTimeValueMapProducer : public edm::global::EDProducer<> {
 DEFINE_FWK_MODULE(TrackTimeValueMapProducer);
 
 namespace {
-  constexpr float fakeBeamSpotTimeWidth = 0.175f; // ns
+  constexpr float fakeBeamSpotTimeWidth = 0.300f; // ns
```

Comment:
is this documented anywhere? 
even though the name is "fake", it's supposedly not an arbitrary number.